File: app/src/main/java/com/example/mytemplate/utils/UserMessage.java
package com.example.mytemplate.utils;

import android.content.Context;
import android.widget.Toast;

public class UserMessage {

    public static void defaultMessage(Context context, String message){
        if(context == null) return;

        Toast.makeText(context, message, Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/com/example/mytemplate/utils/UserMessage.kt
package com.example.mytemplate.utils

import android.content.Context
import android.widget.Toast

class UserMessage(private val context: Context?) {

    fun defaultMessage(message: String?) {
        context?.let {
            Toast.makeText(context, message, Toast.LENGTH_SHORT).show()
        }
    }
}